sdk/textanalytics/Azure.AI.TextAnalytics/README.md

```diff
@@ -837,7 +825,7 @@ This project has adopted the [Microsoft Open Source Code of Conduct][code_of_con
 [custom_subdomain]: https://docs.microsoft.com/azure/cognitive-services/authentication#create-a-resource-with-a-custom-subdomain
 [DefaultAzureCredential]: https://github.com/Azure/azure-sdk-for-net/tree/main/sdk/identity/Azure.Identity/README.md
 [logging]: https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/core/Azure.Core/samples/Diagnostics.md
-[data_limits]: https://docs.microsoft.com/azure/cognitive-services/text-analytics/overview#data-limits
+[data_limits]: https://docs.microsoft.com/azure/cognitive-services/text-analytics/concepts/data-limits?tabs=version-3
```

Comment:
ohh nice, i'll update this too

Reply:
the link still works in case you want to leave it. It just makes the user do an extra jump

Reply:
Ohh you made me realize I have this link everywhere. Might as well update it everywhere too :P